packages/components/src/calendar-react-day-picker/index.tsx

```diff
@@ -0,0 +1,210 @@
+import { useMemo, useState, useCallback } from 'react';
+import { DayPicker } from 'react-day-picker';
+import { useControlledValue } from './utils';
+import type { DateCalendarProps, DateRangeCalendarProps } from './types';
+
+import './styles.scss';
+
+const BASE_CLASSNAME = 'a8c-components-calendar';
+
+const useCommonProps = ( {
+	numberOfMonths,
+	locale,
+}: {
+	numberOfMonths: number;
+	locale: DateCalendarProps[ 'locale' ];
+} ) => {
+	const commonProps = useMemo( () => {
+		const localeCode = locale?.code ?? 'en-US';
+
+		return {
+			animate: true,
+			// Only show days in the current month
+			showOutsideDays: false,
+			// Hide week number column
+			showWeekNumber: false,
+			// Show weekdays row
```

Comment:
Could we support `weekStartsOn`? https://daypicker.dev/docs/localization#first-date-of-the-week
This is not a big requirement, but may be a good one to support from the get go.

Reply:
My preference is to add props only if we know we need them. hTe starting day of the week already updates accordingly when passing a `locale` prop, and therefore I didn't think it would be necessary to expose that prop.

Do you already have an example of why it would be helpful to expose now?

Reply:
Let me double check with the team, but I believe `locale` is probably enough.

Reply:
Checked with the team, not exposing it is fine, we don't have a use case for it at the moment.

Reply:
Given that @chihsuan also requested it and mentioned that WP has a dedicated setting, I opted for exposing the prop